### PR TITLE
Made gift dialog translatable

### PR DESCRIPTION
**Pull request by ghost**
Addressing a bit beyond _#10 in the opening post of
https://github.com/HabitRPG/habitrpg/issues/4522

Screenshot of before-after-proof: http://oi57.tinypic.com/11c9ldi.jpg

### CODE REVIEW

File: website/views/shared/modals/members.jade [hunk 1]
```diff
@@ -62,14 +62,14 @@ script(type='text/ng-template', id='modals/private-message.html')
 
 script(type='text/ng-template', id='modals/send-gift.html')
   .modal-header
-    h4 Send Gift to {{::profile.profile.name}}
+    h4=env.t('sendGiftHeading', {name: "{{profile.profile.name}}"})
```
Comment (crookedneighbor) [inline website/views/shared/modals/members.jade:65]: You've removed the one-time binding symbol `::`

Comment (ghost) [inline website/views/shared/modals/members.jade:65]: I'm sorry about this. This issue was a bit tricky for me, so I appreciate your help @crookedneighbor 
May I change this back and push the changes now?

Comment (crookedneighbor) [inline website/views/shared/modals/members.jade:65]: I noticed some other places that could use some one time bindings, so I just added them in: https://github.com/HabitRPG/habitrpg/commit/fce9d39f19cb3080e6bca4e1b96e5a7d9ca20ee6

**Comment by crookedneighbor** [comment]
Notated toward your next tier.

**Comment by GitHubSphinx** [comment]
@crookedneighbor Thank you and please excuse me, but I haven't quite understood whether I should bring the one-time binding symbol :: back?

**Comment by ghost** [comment]
@crookedneighbor Thank you very much for your kind help! Have a wonderful day full of inspiration!

**Pull request merged**